allocations/allocations_api.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import json
+from django.conf import settings
+import logging
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+
+logger = logging.getLogger('default')
+
+class BALANCE_SERVICE_API:
```

Comment:
It's not typical to name Pyton classes LIKE_THIS - that convention is reserved for constants. How about `BalanceServiceClient`?